# Data Structure for Saving Coproducts

std_msgs/Float32MultiArray data
string msglabel
int16 sceneid
int16 imgid